numpy/lib/function_base.py

```diff
@@ -4819,6 +4819,13 @@ def trapz(y, x=None, dx=1.0, axis=-1):
     >>> np.trapz([1,2,3], dx=2)
     8.0
 
+    Numerical integration of a simple function on a finite interval:
+    
+    >>> x=np.linspace(0,1)
+    >>> y=x**2
+    >>> abs( np.trapz(y, x) - 1/3 ) < 1e-3
+    True
```

Comment:
Thanks, please make sure to use PEP 8 formatting?

My feeling would be that just printing the result is easier to read, the abs and comparison just make things harder?  The text can say that 1/3 is expected.

(Or use `np.isclose` probably on two lines of code with a descriptive variable name)

Reply:
I simplified a bit of wording and incorporated your suggestions. I was worried showing a 16-digit result may be confusing to some but looking back I see it's done elsewhere already.

Reply:
Sure - for some reason I thought `num=100` was the default, but apparently it's 50. I used `num=50` following the default, but made it explicit. I could have believed either; since it's O(h^2) convergence, and everything is order one, I'd believe an error of  about 0.01**2 or 0.02**2.